bundle/manifests/node-observability-operator.clusterserviceversion.yaml

```
@@ -339,12 +339,15 @@ spec:
                   seccompProfile:
                     type: RuntimeDefault
               - args:
-                - --leader-elect
                 - --health-probe-bind-address=:8081
                 - --metrics-bind-address=127.0.0.1:8080
                 - --leader-elect
+                - --zap-log-level=$(LOG_LEVEL)
```

Doesn't `$()` evaluate expressions? Does this work? Maybe you meant this.

```suggestion
                - --zap-log-level=${LOG_LEVEL}
```

---

I think `$()` here is correct - [docs](https://kubernetes.io/docs/tasks/inject-data-application/define-command-argument-container/#use-environment-variables-to-define-arguments)

> Note: The environment variable appears in parentheses, "$(VAR)". This is required for the variable to be expanded in the command or args field.